www/index.md

```diff
@@ -31,6 +31,19 @@ IE11 compatible
 By removing these arbitrary constraints htmx completes HTML as a 
 [hypertext](https://en.wikipedia.org/wiki/Hypertext)
 
+## HTML fragments over the wire
+
+Serializing data to JSON on the server and parsing the JSON on the client is the way 
+many frameworks (Angular, React Vue) work today.
+
+But maybe there is a simpler way?
+
+htmx transfers server-rendered HTML fragments and thus simplifies how modern web applications
+get developed.
+
+htmx extends HTML while staying true to its declarative nature to enable loading and submitting 
```

Comment:
```suggestion
htmx extends HTML while staying true to its declarative nature to enable dynamic
user interfaces just by loading and submitting HTML fragments - AKA "HTML
fragments over the wire".
```

This part didn't seem to explain why loading HTML fragments is useful, so here's a suggestion.

Other than that, LGTM

Reply:
Yes, thank you!